artman/tasks/python_grpc_tasks.py

```diff
@@ -96,45 +100,49 @@ def _extract_base_dirs(self, proto_file):
                     return os.path.sep.join(pkg.split('.'))
             return ''
 
-    def _transform(self, pkg, sep, common_protos):
+    def _transform(self, pkg, sep, common_protos, api_name, api_version):
         """Transform to the appropriate proto package layout.
 
         Works with arbitrary separator (e.g., '/' for import statements,
         '.' for proto package statements, os.path.sep for filenames)
         """
+        # Split the proto path into a list, and move the `.proto` extension
+        # off to the side for now.
         if sep != '.' and pkg.endswith('.proto'):
-            dotted = pkg[:-6].replace(sep, '.')
+            dotted = pkg[:-6].replace(sep, '.').split('.')
             suffix = '.proto'
         else:
-            dotted = pkg.replace(sep, '.')
+            dotted = pkg.replace(sep, '.').split('.')
             suffix = ''
 
-        # Sanity check: Do not transform common protos.
-        for common_pkg in common_protos:
```

Comment:
Also, do we really want to no-op for unversioned, *non*-common protos (e.g., `google.appengine.legacy`; I do not know if there are any examples in actual current codegen use), or do we still want to do the "cloud" transformation below?